src/components/Cards/Cards.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import themeGet from 'extensions/themeGet';
+import Card from './Card';
+
+const Cards = styled.div`
+  display: flex;
+
+  & > * {
+    margin: ${themeGet('spacing.medium')};
+    border-radius: 5px;
+    box-shadow: ${themeGet('shadows.long')};
+  }
+`;
+
+Cards.Connected = Cards.extend`
+  & > * {
```

Comment:
I think these relationships may be better inverted and placed in `Card`'s styling components...

```
${Cards} > & {
  margin: 0;
  / * etc */
}
```

This keeps relevant CSS properties contained in the stylesheet of the component they apply to, and makes the Card component more easily customizable on its own.

Reply:
Re: usage
It would be great if we had a real-world example of where this was going to be used, this could help inform the design decisions.  I like the idea of making Card the primary component, since that makes the usage of a single Card on the page much easier.

I'm not sure about using a child component of Card as a parent of Card on the page, this seems a bit counter-intuitive to me.  Would it perhaps be better to include a separate element e.g. Deck to optionally contain Cards?  The downside to this is that outside of semantics, it doesn't show a relationship between the two components.  In general, I struggle with figuring out how to relate two components, since modeling relationships immediately puts me in an object-oriented state of mind.

Either way using Card as the top-level element seems to be a better solution than the Cards.Card solution, so I will start refactoring towards that goal.

Reply:
Another thought concerning the Header design: having a Card.Header as its own component lets the markdown be more semantic, but doesn't enforce a single header at the top of a card.  That is to say, the developer can include more than one header, and can include them anywhere in the stack of card sections.  I'm not sure that this is much of a concern, since ultimately it's on the developer to use the components appropriately in the first place, but including header title/image props does prevent the developer from using the header in a manner which we didn't intend it to be used.

Reply:
Including a reference to another component as a static property, i.e. `Cards.Group`, doesn't necessitate that component being considered a 'child'; just a convenience reference. For another example, see `Accordion.Group`. We only need to consider which component will be referenced more often. Since a `group : item` relationship is `1 : x, x >= 1`, we can reasonably assume that Card will be referenced most frequently and should be the primary export to save keystrokes for the developer.